src/app/modules/main/item.nim

```diff
@@ -7,7 +7,6 @@ type
     Wallet,
     WalletV2,
     Browser,
-    Timeline,
```

Comment:
> we have an enum which maps status-go chat type to dto chat type and after that in module.nim of this module we have 1:1 mapping dto chat type to the item section type

That's interesting! I guess, it's then prolly indeed easiest to reintroduce `Timeline`...

I do think however, that we want to have this coupling. Here's why:

1. This is a section type enum that shouldn't be affected by the chat type variants and vice versa. We should be able to remove/add `SectionType` variants and not have to deal with this conflict here.
2. It seems inconsistent if we assume `SectionType` maps to `ChatType` because only a subset of these variants do indeed map. For example `Chat` is `0` for both enums, but `Community` is `1` here while `6` in the other.

So I think we should either map everything (**explicitly**) or nothing (and have these not affect each other).

> remove it in status-go that we're sure that dto chat type is 1:1 mapping with types on the status-go side

We can't remove it from status-go, because we don't want to change protocol. All we want to do is remove an app section of our application. Other apps may choose to still serve a timeline (mobile does for example)

> in module.nim of this module we have 1:1 mapping dto chat type to the item section type,

I can't find this. Can you show me the place where we map the chat type to the section type?

Reply:
@PascalPrecht sorry cause I bring some confusions here. :) I jumped into reviewing this from some chat work and missed the thing here. There is no any mapping since we're setting section type in `load` method of the `module.nim`, so all good.

Reply:
Thanks for clarifying!